zuul.d/ansible-test-jobs.yaml

```diff
@@ -22,6 +22,21 @@
       ansible_test_python: 3.8
       container_command: podman
 
+- job:
+    name: ansible-test-sanity-docker-devel
+    parent: ansible-test-sanity-docker
+    required-projects:
+      - name: github.com/ansible/ansible
+        override-checkout: devel
+    voting: false
```

Comment:
we shouldn't do this my default, no one looks at non-voting jobs and it is a waste of resources.  We should keep it voting and projects that don't want to track devel can just remove it now IMO.

This however expose the issue where we are no longer testing against the latest code and makes it more harder to get into compliance closer to release time.

Reply:
If we stop testing `devel`, we will end-up with breaking change in milestone. If we keep `devel` voting, we end up with broken tests on our side. I dislike both. But at this stage, I would prefer to continue to test devel as non-voting, and see if we can report the issues efficiently.